### PR TITLE
KFSPTS-22552 add cookie processor setting to context.xml

### PR DESCRIPTION
**Pull request by jhulslander**
This change allows for better cookie persistence when using eshop

### CODE REVIEW

**Review by chadbower — COMMENTED** [review]
This is the value we tested manually

**Comment by jhulslander** [comment]
Actually, please don't merge this yet

**Comment by jhulslander** [comment]
retest this please

**Comment by nkimble** [comment]
@jhulslander  Based on the Tomcat documentation I saw on-line, by specifying this setting with the value of "none", the same-site cookie attribute will be set and the cookie will always be sent in cross-site requests.  This is the behavior we want, correct?

**Comment by jhulslander** [comment]
@chadbower the issue last night where the navigation pane would start to appear and then disappear.  I am experiencing the problem locally with Chrome as well.  Firefox does does work locally with this configuration.  This configuration will need some tweeks

**Comment by jhulslander** [comment]
@nkimble  the goal of this change was to prevent the jsession and kualisession cookies from being deleted when in the eshop iframes.  With out configuration this cookie configuration change would only affect the eshop integration, which is fairly scoped down.  @chadbower do you have further details to Nancy's question?

**Comment by db483** [comment]
@jhulslander This change looks similar to the fix KualiCo put in place for the Jaegger issue, take a look at the email from Lesley. They have a couple other changes in place.

**Comment by jhulslander** [comment]
As @db483  mentioned, KualiCo had some other changes they implemented for Stevens.  I have added those changes to this PR

File: src/main/java/org/kuali/kfs/kns/util/CookieUtils.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    public static final String AUTH_TOKEN_COOKIE_NAME = "authToken";
+    public static final String FIN_AUTH_TOKEN_COOKIE_NAME = "financialsAuthToken";
+
+    public Cookie createCookie(HttpServletRequest request, String name, String value, String path) {
+        Cookie cookie = new Cookie(name, value);
+        /*
+         * KFSPTS-22552
+         * Original base code was cookie.setSecure(request.isSecure()); 
+         */
+        cookie.setSecure(true);
```
Comment (nkimble) [inline src/main/java/org/kuali/kfs/kns/util/CookieUtils.java:40]: @db483 Do we need to know when "request.isSecure()" would return a false value?

**Review by nkimble — APPROVED** [review]
@db483 and I conversed regarding the logging question off-line. Additional questions will be posed to KualiCo in a subsequent meeting regarding why they choose their fix of the hard setting of the cookie secure value to true. Merging these changes.

**Pull request merged**